src/main/java/teammates/storage/api/StudentsDb.java

```diff
@@ -119,6 +120,23 @@ public StudentAttributes createEntity(StudentAttributes student)
         return createdStudent;
     }
 
+    /**
+     * Creates a batch of students.
+     *
+     * @return the created students
+     * @throws InvalidParametersException if the student is not valid
+     * @throws EntityAlreadyExistsException if the student already exists in the Datastore
+     */
+    @Override
+    public List<StudentAttributes> putEntities(Collection<StudentAttributes> students)
+            throws InvalidParametersException {
+
+        List<StudentAttributes> createdStudents = super.createEntitiesSilent(students);
```

Comment:
`super` keyword is not necessary here?

Reply:
I think you are right; not sure if all of the super keywords in the same class should be removed by your logic though (sorry not too well versed in Java to be confident in this change)